runtime/oti/FFIUpcallThunkGenHelpers.hpp

```diff
@@ -323,4 +362,93 @@ AFI(I_8 *instructionPtr, I_32 sourceRegister, I_32 immediateInteger)
 
 }
 
+/*
+ * Generates a sequence of instructions to copy data from a Java MemorySegment Object to a C struct.
+ * The struct is smaller than or equal to 4096 bytes.
+ */
+static I_32
+generateSmallStructCopyInstructions(I_8 *instructionPtr, I_32 structSize, I_32 offsetToParameterArea, I_32 sourceRegister, I_32 destinationRegister)
+{
+	I_32 totalInstructionSize = 0;
+
+	const I_32 numLargeCopies = structSize / MAX_MVC_COPY_LENGTH;
+	const I_32 residue = structSize % MAX_MVC_COPY_LENGTH;
+
+	// Generate upto 16 MVC instructions to copy up 4096 bytes. Each MVC instruction in this sequence will copy 256 bytes.
+	for(int i = 0; i < numLargeCopies; i++) {
```

Comment:
Nit. Space between `for` and `(` please.

Reply:
Fixed it here: 41862518458